### ISSUE TITLE
Request FlubuCore.GlobalTool to support .NET 5

### ISSUE THREAD

**Issue by yang-xiaodong** [dotnetcore/FlubuCore#340]
Hello @mzorec ,

When I use .NET 5 as the build version in travis-ci, it will not work properly, so do we have plans to make FlubuCore.GlobalTool supported in .NET 5

Error info :
```
You can invoke the tool using the following command: flubu
Tool 'flubucore.globaltool' (version '5.1.1') was successfully installed.
The command "dotnet tool install --global FlubuCore.GlobalTool --version 5.1.1" exited with 0.
0.00s$ flubu build tests
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '3.0.0' was not found.
  - The following frameworks were found:
      5.0.1 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]
You can resolve the problem by installing the specified framework and/or SDK.
```

**Comment by mzorec** [comment]
Yes we plant to support .net 5 in Flubu 6.0 It should be released in few days. 

Meanwhile you can probably add to travis something like this:

before_script:
- sudo apt-get install dotnet-sdk-3.0

**Comment by mzorec** [comment]
Support for .net5.0 is available in 6.0.0-preview4. 

Version 6.0.0 should be available in couple of days when documentation is updated and 6.0.0-preview4 is tested.
